plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -963,6 +965,57 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
          EOS_THROW( extract_genesis_state_exception, "extracted genesis state from blocks.log" );
       }
 
+      std::optional<chain_id_type> chain_id;
+      if( options.count("snapshot-to-json") ) {
```

Comment:
Can `snapshot-to-json` be changed to something like `snapshot-to-json-dir`. Initially I thought `snapshot-to-json` is a flag.